### PR TITLE
GCP OVNKubernetes ipi workflow

### PR DESCRIPTION
**Pull request by davidvossel**
I'm working on hypershift's KubeVirt platform and I need an ipi workflow for setting up GCP with OVN.

### CODE REVIEW

**Comment by nunnatsa** [comment]
Hi @smarterclayton, @wking, @vrutkovs, @abhinavdahiya, @deads2k, @staebler - could you please approve this PR?

File: ci-operator/step-registry/ipi/gcp/ovn/OWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+approvers:
+- abhinavdahiya
```
Comment (stbenjam) [inline ci-operator/step-registry/ipi/gcp/ovn/OWNERS:2]: I’d suggest having more than one approver from whoever is going go to own the jobs that will use this workflow. 

Why just Abhinav?

File: ci-operator/step-registry/ipi/gcp/ovn/ipi-gcp-ovn-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: ipi-gcp-ovn
+  steps:
+    pre:
+    - chain: ipi-conf-gcp
+    - ref: ovn-conf
+    - chain: ipi-install
+    post:
```
Comment (stbenjam) [inline ci-operator/step-registry/ipi/gcp/ovn/ipi-gcp-ovn-workflow.yaml:8]: Why not the gcp post workflow ?

**Comment by stbenjam** [comment]
I think I’d prefer to see you add the jobs that use this workflow in this PR so we can see a rehearse and understand how it’s going to be used.

**Pull request closed without merging**